src/storage/storage.ts

```diff
@@ -699,6 +699,27 @@ export class Storage implements IStorage {
     }
   }
 
+  /**
+   * Get account path xprivkey if available.
+   *
+   * @param {string} pinCode
+   * @returns {Promise<string>}
+   */
+  async getAcctPathXPrivKey(pinCode: string): Promise<string> {
+    const accessData = await this._getValidAccessData();
+    if (accessData.acctPathKey === undefined) {
```

Comment:
Should we also check for `null`?

Reply:
We can but `IWalletAccessData` does not allow null as a value for this.